docs/src/frontend/index.md

```diff
@@ -1060,3 +1060,26 @@ ask.done() => null
 ```
 
 Read the [Interaction and Independence](##tut-8) section the Rock, Paper, Scissors tutorial for a longer use case example of the `{!js} ask` object.
+
+---
+@{ref("js", "launchToken")}
+```js
+launchToken(accCreator: Account, name: string, sym: string, opts?: LaunchTokenOpts) => Promise<object>
+```
+
+Launches a new token with the given `{!js} name` and unit symbol `{!js} sym`.
+Launched on the network by `{!js} accCreator`.
+
+Possible options to provide in `{!js} opts` include:
++ `{!js} decimals` The number of digits to use after the decimal point when displaying the asset.
+The default is the same as the network token.
++ `{!js} supply` The total number of atomic token units to create.
+The default is the maximum possible on the network.
++ `{!js} url` Algorand only. A URL where more information about the asset can be retrieved.
+The default is no url.
++ `{!js} metadataHash` Algorand only. A hash of some metadata that is relevant to your asset.
+The default is no metadata hash.
++ `{!js} clawback` Algorand only. Address that can claw back holdings of the token.
```

Comment:
Put in a separate section that says something like "These options are available on Algorand:" ....